src/algorithm/pso.cpp

```diff
@@ -565,6 +567,51 @@ void pso::initialize_topology__adaptive_random( std::vector< std::vector<int> >
 }
 
 
+/**
+ * @brief Arranges particles in a 2D lattice with alternatively staggered rows, where each interacts with its 6 nearest neighbours.
+
+*/
+const int hexagonal_neighb_diff_even[6][2] = { {-1,-1}, {0,-1}, {-1,0}, {1,0}, {-1,1}, {0,1} };
+const int hexagonal_neighb_diff_odd[6][2] = { {0,-1}, {1,-1}, {-1,0}, {1,0}, {0,1}, {1,1} };
+
+void pso::initialize_topology__hexagonal( std::vector< std::vector<int> > &neighb ) const
+{
+        int swarm_size = neighb.size();
+        int     cols, rows;             // lattice structure
```

Comment:
There's no reason to align variable names in the column with multiple spaces, use a single space.